ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationPeriodValidationIT.scala

```diff
@@ -30,6 +42,44 @@ class CommandDeduplicationPeriodValidationIT extends LedgerTestSuite {
     }
   })
 
+  test(
+    "NegativeDeduplicationDuration",
+    "Submission with negative deduplication durations are rejected",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(ledger, party)) =>
+    val deduplicationPeriod = DeduplicationPeriod.DeduplicationDuration(
+      DurationConversion.toProto(Duration.ofSeconds(-1))
+    )
+    assertFailedRequest(
+      ledger,
+      party,
+      deduplicationPeriod,
+      "Requests with a deduplication period represented by a negative duration",
+      "The submitted command has a field with invalid value: Invalid field deduplication_period: Duration must be positive",
```

Comment:
Not sure if it can be done here but nit:

```suggestion
      "Invalid field in the submitted command: Invalid field deduplication_period: Duration must be positive",
```

Reply:
This is not really changeable here, we assert on the error message and that's what's generated. Will use named arguments.